### DISCUSSION TITLE
Using wiki.js with httpd centos9

### DISCUSSION THREAD

**Discussion by gearsovwar** [requarks/wiki#6603] [Help / Questions] [answered]
Hope everyone is well,

I am trying to setup wiki.js to use httpd virtual directories. I have created the config file in /etc/httpd/servers-enabled and created the sysmlink. I have setup the directories in /var/www/example.com but my understanding is httpd by default loads the index.html from that directory. im unsure of how to get wiki.js running with this method. I can configure a non standard port and it will work just fine but i am trying to host multiple sites on port 80 by using virtual directories.

Thanks again im still fairly new to apache/httpd and hosting with it sorry if i missed anything.

**Comment by NGPixel** [comment, marked as answer]
Wiki.js is its own server and doesn't work with virtual directories. You need to use a reverse proxy in your config to point your domain/subdomain to the Wiki.js port.

See https://httpd.apache.org/docs/2.4/howto/reverse_proxy.html
